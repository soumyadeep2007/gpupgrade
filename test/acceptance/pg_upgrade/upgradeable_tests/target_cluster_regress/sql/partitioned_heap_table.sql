-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Validate that the upgradeable objects are functional post-upgrade
--------------------------------------------------------------------------------

SELECT * FROM p_basic;

SELECT * FROM p_add_partition_test;

SELECT * FROM p_add_list_partition_test;

SELECT * FROM p_split_partition_test;

SELECT id, age FROM p_subpart_heap_1_prt_partition_id_2_prt_subpartition_age_first;
SELECT id, age FROM p_subpart_heap_1_prt_partition_id_2_prt_subpartition_age_second;
SELECT id, age FROM p_subpart_heap;

SELECT c, d FROM p_dropcol WHERE a=10;
