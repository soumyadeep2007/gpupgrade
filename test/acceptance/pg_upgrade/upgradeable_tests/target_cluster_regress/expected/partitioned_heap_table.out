-- Copyright (c) 2017-2021 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

--------------------------------------------------------------------------------
-- Validate that the upgradeable objects are functional post-upgrade
--------------------------------------------------------------------------------

SELECT * FROM p_basic;
 id | name 
----+------
 1  | Jane 
 2  | John 
(2 rows)

SELECT * FROM p_add_partition_test;
 a | b 
---+---
 1 | 1 
 2 | 1 
 1 | 2 
 1 | 3 
(4 rows)

SELECT * FROM p_add_list_partition_test;
 a | b 
---+---
 1 | 1 
 2 | 1 
 1 | 2 
 1 | 3 
(4 rows)

SELECT * FROM p_split_partition_test;
 a | b 
---+---
 1 | 1 
 2 | 2 
 3 | 3 
 4 | 4 
 5 | 5 
(5 rows)

SELECT id, age FROM p_subpart_heap_1_prt_partition_id_2_prt_subpartition_age_first;
 id | age 
----+-----
 1  | 10  
(1 row)
SELECT id, age FROM p_subpart_heap_1_prt_partition_id_2_prt_subpartition_age_second;
 id | age 
----+-----
 2  | 20  
(1 row)
SELECT id, age FROM p_subpart_heap;
 id | age 
----+-----
 1  | 10  
 2  | 20  
(2 rows)

SELECT c, d FROM p_dropcol WHERE a=10;
 c  | d  
----+----
 10 | 10 
 10 | 10 
(2 rows)
